packages/react-aria-components/test/AriaAutocomplete.test-util.tsx

```diff
@@ -877,7 +877,7 @@ export const AriaAutocompleteTests = ({renderers, setup, prefix, ariaPattern = '
           let {getByRole, getAllByRole} = (renderers.subdialogs!)();
           let menu = getByRole('menu');
           let options = within(menu).getAllByRole('menuitem');
-          expect(options[1]).toHaveAttribute('aria-haspopup', 'dialog');
+          expect(options[1]).toHaveAttribute('aria-haspopup', 'menu');
```

Comment:
now we don't know if the submenu will be a dialog or not, or rather, they are all dialogs? I didn't notice any announcement differences in real screen readers